src/component/trade/form.ts

```diff
@@ -470,16 +474,16 @@ export class TradeForm extends LitElement {
           <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
             <path d="M6 4H20V6H22V12H14V14H20V16H16V18H14V20H2V8H4V6H6V4ZM8 10H10V8H8V10Z" fill="#FF6868" />
           </svg>
-          <span class="text_error">Sorry but trade amount is too small to execute order.</span>
+          <span class="text_error">${i18n.t('trade.error.tooSmallForTwap')}</span>
         </span>
       `;
     }
   }
 
   infoTwapSlippageTemplate(assetSymbol: string) {
     return html` ${choose(this.tradeType, [
-        [TradeType.Sell, () => html` <span class="label">Minimum received:</span>`],
-        [TradeType.Buy, () => html` <span class="label">Maximum sent:</span>`],
+        [TradeType.Sell, () => html` <span class="label">${i18n.t('trade.settings.twapSummary.minReceived')}</span>`],
+        [TradeType.Buy, () => html` <span class="label">${i18n.t('trade.settings.twapSummary.maxSpent')}:</span>`],
```

Comment:
@nohaapav do i understand this one correctly?

Reply:
yes